src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -49,9 +49,8 @@
 import hudson.model.Computer;
 import hudson.model.JDK;
 import hudson.model.Run;
-import hudson.plugins.sonar.action.BuildSonarAction;
-import hudson.plugins.sonar.action.ProjectSonarAction;
-import hudson.plugins.sonar.action.UrlSonarAction;
+import hudson.plugins.sonar.action.SonarMarkerAction;
+import hudson.plugins.sonar.action.SonarBuildBadgeAction;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'hudson.plugins.sonar.action.SonarBuildBadgeAction'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)